src/array/reducer/remove-control.spec.ts

```diff
@@ -17,6 +17,7 @@ describe(`form group ${removeControlReducer.name}`, () => {
     expect(resultState.value).toEqual([INITIAL_FORM_ARRAY_VALUE[1]]);
     expect(resultState.controls[1]).toBeUndefined();
     expect(resultState.controls[0].id).toEqual(`${FORM_CONTROL_ID}.0`);
+    expect(resultState.isDirty).toEqual(true);
```

Comment:
There is indeed a test missing for setting the `isDirty` flag. However, instead of implicitly testing it in here, there should be a separate test. In any case, please remove these changes from the PR so that it only contains code related to the feature. I will add the missing test and fix the typo directly in the source. Thanks for pointing this out.